dbt/adapters/athena/constants.py

```diff
@@ -1,3 +1,41 @@
 from dbt.events import AdapterLogger
 
-LOGGER = AdapterLogger("Athena")
+DEFAULT_THREAD_COUNT = 4
+DEFAULT_RETRY_ATTEMPTS = 3
+DEFAULT_POLLING_INTERVAL = 5
+DEFAULT_SPARK_COORDINATOR_DPU_SIZE = 1
+DEFAULT_SPARK_MAX_CONCURRENT_DPUS = 2
+DEFAULT_SPARK_EXECUTOR_DPU_SIZE = 1
+DEFAULT_CALCULATION_TIMEOUT = 43200  # seconds = 12 hours
+SESSION_IDLE_TIMEOUT_MIN = 10  # minutes
+
+DEFAULT_SPARK_PROPERTIES = {
```

Comment:
@nicor88 Just wanted to provide an update. @sankeerthnagapuri did a great work in this [PR](https://github.com/Avinash-1394/dbt-athena-1/pull/1) to my branch to allow providing more spark properties. He's also tested it to be working. Would it be possible to include this in a release any time soon?

Reply:
@nicor88 @svdimchenko @Jrmyy Pinging again to see if this can be included in an upcoming release.

Reply:
@Avinash-1394 I've approved the PR. Let's try to merge it and release then